.changeset/nine-dancers-reflect.md

```diff
@@ -0,0 +1,11 @@
+---
+"@hopper-ui/icons": minor
+---
+
+**Added**
+
+| SVG Name                             |
+| ------------------------------------ |
+| folder-16.svg                           |
```

Comment:
that would be the release notes for svg-icons. 

you also need release notes for the icon package

Reply:
icons only release the FolderIcon icon